index.bs

```diff
@@ -8115,6 +8181,161 @@ JavaScript Array values.
 </div>
 
 
+<h4 id="js-async-iterable">Async iterable — async iterable&lt;|T|&gt;</h4>
+
+IDL <a lt="async iterable type">async iterable&lt;|T|&gt;</a> values are represented by a JavaScript
+object, a JavaScript method, and a flag indicating whether the method is expected to produce a
+<code>sync</code> or <code>async</code> iterator.
```

Comment:
Can we give these fields proper `<dfn>`s? We're not consistent about this in general but I'd like to be here since this is one of the more complex fields we have. #idl-callback-interface has an example where it defines `callback context` (although it tries to be language-agnostic, which is probably not right).

Something like

```html
In the JavaScript binding, IDL [=async iterable type|async iterable=] values are represented by a [=struct=] with the following [=items=]:

* <dfn for="JS async iterable">object</dfn>, a JavaScript value
* <dfn for="JS async iterable">method</dfn>, a JavaScript value
* <dfn for="JS async iterable">type</dfn>, either "<code>sync</code>" or "<code>async</code>"
```

and then reference those struct items throughout.